doc/source/user_guide/changelog/v0.4.1.rst

```diff
@@ -30,7 +30,7 @@ Maintenance
 - improved variable naming for clarity and in line with common usage (#211)
 - add tests that require an active NSIDC Earthdata session (#209)
 - update tracking metrics and limit traffic action to parent repo (#221)
-- remove extra code block frome xample notebook (#225)
+- remove extra code block from example notebook (#225)
```

Comment:
The release date of v0.4.1 at L3 should be 1 Dec 2021, not 1 Oct 2021 :slightly_smiling_face:

Reply:
Yes, but it's just the changelog, so no biggie really.